packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -19,6 +19,24 @@ import { isDesktop, isMobile } from '@deriv/shared/utils/screen';
 import { Localize, localize } from '@deriv/translations';
 import { isDeepEqual } from '@deriv/shared/utils/object';
 
+const account_opening_reason_list = [
+    {
+        text: localize('Hedging'),
```

Comment:
these won't get localized because it's in a constant and not an array, and it gets initialised before localize is populated with all our languages.

Reply:
TIL moment. Thanks.